test/models/review_test.rb

```diff
@@ -1,7 +1,58 @@
 require "test_helper"
 
 describe Review do
-  # it "does a thing" do
-  #   value(1+1).must_equal 2
-  # end
+  before do
+    @product = products(:begonia)
+    @review = Review.create!(rating: 4, comment: "I love the pretty flowers!", product_id: @product.id)
+  end
+  
+  describe 'relations' do
+    it 'belongs to a product' do
+      expect(@review.product_id).must_equal @product.id
+    end
+  end
+  
+  describe 'validations' do
+    let(:product) { products(:begonia) }
+    let(:review) { Review.create!(rating: 4, comment: "I love the pretty flowers!", product_id: @product.id) }
```

Comment:
I think the `before do`, since it's on the wrapping describe block, applies to this describe block as well. Which means you could just use `@product` and `@review` from the `before do` in place of `let(:product)` and `let(:review`.